rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -115,14 +116,17 @@ install -m 644 files/config.json.example %{buildroot}/%{_datadir}/%{name}/
 /srv/salt/fpf*
 %doc README.md
 %license LICENSE
-
+%{_userunitdir}/securedrop-user-dom0-rpm-key-reimport.service
 
 %post
 find /srv/salt -maxdepth 1 -type f -iname '*.top' \
     | xargs -n1 basename \
     | sed -e 's/\.top$$//g' \
     | xargs qubesctl top.enable > /dev/null
 
+# rpm key reimport
+systemctl --global enable securedrop-user-dom0-rpm-key-reimport.service ||:
```

Comment:
Does this need to be a systemd service if it's a one-time operation? could it just be an `rpm --import` for the key in %post?

Reply:
See my comments in the PR body

Reply:
(More detail: when installing an rpm, there's a transaction lock on the rpm database, so you can't do rpm db changes in %post)